application/dash_application/utility/plot_builder.py

```diff
@@ -242,6 +242,36 @@ def reshape_call_ready_df(df, projects, references, tissue_preps, sample_types,
     return df
 
 
+def is_empty_plot(trace_list) -> bool:
+    """
+    Check if any traces will be drawn on the graph
+
+    Args:
+        trace_list: The list of traces. Can be dict or plotly graph objects
+
+    Returns:
+
+    """
+    for t in trace_list:
+        try:
+            if t['x']:
+                return False
+        # Pandas Series has: The truth value of a Series is ambiguous.
+        except ValueError:
```

Comment:
It would be a lot nice if this could check if the value is an instance of `Series` rather than catching an exception that might have other reasons for existing.

Reply:
Thanks for the push. I found a cleaner way of doing this. The hard thing about duck typing is that a duck can quack in so many different and overlapping ways.